service/src/main/scala/org/apache/celeborn/server/common/http/HttpUtils.scala

```diff
@@ -34,4 +55,29 @@ object HttpUtils {
       }
     (url.getPath, parameter)
   }
+
+  def handleRequest(
+      service: HttpService,
+      path: String,
+      parameters: Map[String, String]): (String, Set[HttpEndpoint]) = {
+    var httpEndpoints: Set[HttpEndpoint] = null
+    if (service.serviceName == Service.MASTER) {
+      httpEndpoints = masterEndpoints
+    } else {
+      httpEndpoints = workerEndpoints;
+    }
+    (
+      httpEndpoints.find(endpoint => endpoint.path == path).orElse(Some(Invalid)).get.handle(
+        service,
+        parameters),
+      httpEndpoints)
+  }
+
+  def help(service: String, httpEndpoints: Set[HttpEndpoint]): String = {
+    val sb = new StringBuilder
+    sb.append(s"${Invalid.meaning(service)} Available API providers include:\n")
+    httpEndpoints.foreach(endpoint =>
+      sb.append(s"${endpoint.path}: ${endpoint.meaning(service)}\n"))
```

Comment:
` sb.append(s"${endpoint.path.padTo(30, "")} ${endpoint.meaning(service)}\n"))`
Can pad URI to a same length to make it more tidy. 
Can the desc part of URI be made into a new line as many characters as possible?